spinoso-time/src/time/tzrs/offset.rs

```diff
@@ -8,6 +10,17 @@ use tz::timezone::{LocalTimeType, TimeZoneRef};
 use tzdb::local_tz;
 use tzdb::time_zone::etc::GMT;
 
+#[derive(Debug, Clone)]
+pub struct TzStringError(String);
```

Comment:
Resolving because of https://github.com/artichoke/artichoke/pull/1927#discussion_r910142415 <-- defer decision later, but let me know if we want to tackle this one now.